### DISCUSSION TITLE
Enviroment Variable Showing up after Compile

### DISCUSSION THREAD

**Discussion by PRigter** [parcel-bundler/parcel#5403] [General] [answered]
Hi All!

What the heck I'm doing wrong? Enviroment Variable is showing off it's value after compiling. 

I'm using **dovenv npm module**
`require('dotenv').config()`
![image](https://user-images.githubusercontent.com/30890518/100554151-b2b4c880-328a-11eb-96cf-4afb921b9fa5.png)

**.env file is on root folder**
![image](https://user-images.githubusercontent.com/30890518/100554072-44700600-328a-11eb-9cb8-f93001b98343.png)

`const API_KEY = process.env.API_KEY`

**parcel build, is outputting to a "prod" folder (for app organization)**
![image](https://user-images.githubusercontent.com/30890518/100554270-73d34280-328b-11eb-8776-07a3b5814dcb.png)

But on the .js file automatically created, the Enviroment Value is visible!
![image](https://user-images.githubusercontent.com/30890518/100554223-2d7de380-328b-11eb-9eeb-53d47f232bad.png)

The intention is deploying the prod folder, what am I doing wrong?
It's supposed to behave like this?

Many thanks!
Stay safe 😷

**Comment by mischnic** [comment, marked as answer]
You probably want `--target node` (the default is browser, where env variables are inlined)

**Reply by PRigter** [reply]
Thanks a lot @mischnic !!
Awesome That was it! 

Thanks again!!
